src/scripts/main.js

```diff
@@ -1,3 +1,9 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+for (const row of table.rows) {
+  const clone = row.cells[1].cloneNode(true);
+
+  row.cells[4].before(clone);
```

Comment:
it's always better not use `magic numbers`
Create constants for both numbers (`1` and `4`)